spec/vcr_setup.rb

```diff
@@ -0,0 +1,35 @@
+require 'vcr'
+require 'active_support/core_ext/string'
+
+def filter_body_field(vcr_config, part, field)
+  vcr_config.filter_sensitive_data("<#{field.underscore.upcase}>") do |interaction|
+    body_match = interaction[part].body.match(/"#{Regexp.quote(field)}":\s*"([^"]+)"/)
+    body_match && body_match.captures && body_match.captures.first
+  end
+end
+
+def filter_header(vcr_config, part, placeholder, header)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)